extensions/forked-extensions/src/operation.ts

```diff
@@ -134,13 +133,16 @@ class Operation {
       title: "Copy Logs",
       onAction: () => Clipboard.copy([title, message].join("\n")),
     };
-    return this.showToast({
-      title,
-      message,
-      style: options?.style ?? Toast.Style.Failure,
-      primaryAction: options?.primaryAction ?? copyLogsAction,
-      secondaryAction: options?.primaryAction ? copyLogsAction : undefined,
-    });
+    return this.showToast(
+      {
+        title,
+        message,
+        style: options?.style ?? Toast.Style.Failure,
+        primaryAction: options?.primaryAction ?? copyLogsAction,
+        secondaryAction: options?.primaryAction ? copyLogsAction : undefined,
+      },
+      true, // Create a new toast instance to ensure the `primaryAction.onAction` works as expected.
+    );
```

Comment:
The toast-related bug is weird. I cannot use `primaryAction.onAction` on an existing toast. So I added a workaround to spawn a new toast instance for the `showFailureToast` method.

If you are interested in the bug, you can remove the boolean value `true` from the `showToast` method and observe the issue.

The reproduce steps would be:

1. Go to `src/git.ts`, replace the `rev-parse` with`bug-rev-parse` to let the method throw an exception
2. Run the dev mode and open the extension, it will prompt an error
3. Open the error toast and press enter to run the `Copy Logs` action
4. You will notice that nothing happens; the clipboard copies nothing